src/main/resources/site/parts/statistics/statistics.html

```diff
@@ -2,6 +2,25 @@
     <div class="row">
         <div class="col-12">
             <h1 data-th-if="${title}" data-th-text="${title}" />
+            <div data-th-if="${statistic}" class="statistic">
+                <span class="shortName" data-th-text="${statistic.shortName}" />
```

Comment:
fint å legge til data-th-if for feltene som skal vises :)

Reply:
`shortName` and `name` should always be present. but i will add _data-th-if_ nevertheless!